draft-ietf-quic-http.md

```diff
@@ -2315,17 +2324,18 @@ SETTINGS_HEADER_TABLE_SIZE:
 : See [QPACK].
 
 SETTINGS_ENABLE_PUSH:
-: This is removed in favor of the MAX_PUSH_ID which provides a more granular
-  control over server push.
+: This is removed in favor of the MAX_PUSH_ID frame which provides a more
+  granular control over server push.
 
 SETTINGS_MAX_CONCURRENT_STREAMS:
 : QUIC controls the largest open Stream ID as part of its flow control logic.
-  Specifying SETTINGS_MAX_CONCURRENT_STREAMS in the SETTINGS frame is an error.
+  Specifying the SETTINGS_MAX_CONCURRENT_STREAMS parameter in the SETTINGS frame
+  is an error.
 
 SETTINGS_INITIAL_WINDOW_SIZE:
 : QUIC requires both stream and connection flow control window sizes to be
-  specified in the initial transport handshake.  Specifying
-  SETTINGS_INITIAL_WINDOW_SIZE in the SETTINGS frame is an error.
+  specified in the initial transport handshake.  Specifying the
+  SETTINGS_INITIAL_WINDOW_SIZE parameter in the SETTINGS frame is an error.
```

Comment:
Are we using "parameter" now?  I seem to recall that h2 uses "setting".  And SETTINGS_INITIAL_WINDOW_SIZE isn't a defined parameter, so you might want to say "Specifying a setting with an identifier of `<insert numeric value>` (the identifier for SETTINGS_INITIAL_WINDOW_SIZE in HTTP/2) must be treated as a connection error of type `<blah>`."

Same for the concurrent stream limit.

Reply:
Section 6.5 of RFC 7540:
>    The SETTINGS frame (type=0x4) conveys configuration parameters that
>   affect how endpoints communicate, such as preferences and constraints
>   on peer behavior.  The SETTINGS frame is also used to acknowledge the
>   receipt of those parameters.  Individually, a SETTINGS parameter can
>   also be referred to as a "setting".

But good point that these values aren't defined in this document; I'll reword that.

Reply:
This is now #3954.